README.md

```diff
@@ -91,8 +91,27 @@ DATABASE_NAME="scavenger_hunt.db"
 ```
 Restart the server and it will pick up the settings automatically.
 
-### Option 2: use devcontainers
-TBD
+### Option 2: Use devcontainers (with VSCode)
+If you would like a setup which is similar to a real production environment, use the provided [devcontainer](https://containers.dev). It uses a proper PostgreSQL database server instead of sqlite.
+
+> Before you can use devcontainers make sure you have Docker as well as the VSCode [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) extensions installed. Please refer to the dev container setup documentation on the VSCode page.
+
+Open the cloned repository in [VSCode](https://code.visualstudio.com). Usually it will detect that a `.devcontainer` folder is present and ask you whether you'd like to "Reopen folder to develop in a container". Press "Reopen in Container" and wait.
+
+If VSCode does not prompt you, click on the green area on the bottom left of VSCode and select "Reopen in Container" from the drop down menu at the top.
```

Comment:
The colour is possibly theme or context dependent. The button at bottom left in my VSCode is Blue.